src/main/generated/options/soot/AntTask.java

```diff
@@ -366,6 +370,12 @@ public void setjava_version(String arg) {
                 || arg.equals( "8" )
                 || arg.equals( "1.9" )
                 || arg.equals( "9" )
+                || arg.equals( "1.10" )
+                || arg.equals( "10" )
+                || arg.equals( "1.11" )
+                || arg.equals( "11" )
+                || arg.equals( "1.12" )
+                || arg.equals( "12" )
```

Comment:
if you want to support java 12, we should also update the ASM dependency

Reply:
Haven't you already done this with the Java9 branch? Otherwise, I'm happy to do that.